src/app/components/Common/AlbumPanel/AlbumPanel.jsx

```diff
@@ -144,10 +155,19 @@ class AlbumPanel extends React.Component {
           <span className={classes.subtitle}>{artistName}</span>
 
           {album.attributes.editorialNotes && album.attributes.editorialNotes.standard && (
-            <div className={classes.description}>
-              <span
-                dangerouslySetInnerHTML={{ __html: album.attributes.editorialNotes.standard }} // eslint-disable-line react/no-danger
-              />
+            <div className={classes.description} onClick={this.toggleFullDescription}>
+              {showFullDescription ? (
+                <span
+                  dangerouslySetInnerHTML={{ __html: album.attributes.editorialNotes.standard }} // eslint-disable-line react/no-danger
+                />
+              ) : (
+                <HTMLEllipsis
+                  unsafeHTML={album.attributes.editorialNotes.standard}
+                  maxLine='3'
+                  ellipsisHTML='<i class="read-more">... read more</i>'
```

Comment:
use `ellipsis={<i className='read-more'>... read more</i>}`

Reply:
The library doesn't support this.

Reply:
Apparently it does https://www.npmjs.com/package/react-lines-ellipsis#propsellipsis-node

Reply:
@raphaelvigee 
https://www.npmjs.com/package/react-lines-ellipsis#experimental-html-truncation

"Also, props.ellipsis here only supports plain text, use props.ellipsisHTML is to fully customize the ellipsis style."

If you attempt to use a React node in the elpisis prop of the HTMLEllipsis component it will simply fail to render. I attempted last night, you're welcome to try yourself.

The LinesEllipsis component supports this, however the HTMLEllipsis component does not.

Reply:
<img width="500" alt="Screenshot 2019-05-27 at 12 41 42" src="https://user-images.githubusercontent.com/9306474/58436986-cb806480-807c-11e9-9a4c-41aa15717d15.png">
For reference... [Object object]

Reply:
Humm okay ¯\_(ツ)_/¯